distribution/lib/Standard/Base/0.0.0-dev/src/Internal/Ordering_Helpers.enso

```diff
@@ -37,8 +37,10 @@ type Default_Comparator
     ## PRIVATE
     hash : Number -> Integer
     hash x = Default_Comparator.hash_builtin x
+
```

Comment:
Unnecessary changes.

Reply:
I'm just trying to apply consistent formatting.